ecosystem_cicd_tools/packaging.py

```diff
@@ -194,24 +195,30 @@ def update_assets_in_plugin_dict(plugin_dict, assets, plugin_version=None):
             plugin_dict['link'].split('/')[-2],
             plugin_version
         )
+    wagons_list_copy = sorted(
+        deepcopy(plugin_dict['wagons']),
+        key=lambda d: d['name'])
+
     for asset in assets:
         # Replace the old asset paths with new ones.
         if asset.endswith('.yaml'):
             plugin_dict['link'] = asset
-            continue
-        for wagon in plugin_dict['wagons']:
-            if 'aarch64' in asset:
-                continue
-            if wagon['name'] == REDHAT and 'redhat-Maipo' in asset:
-                if asset.endswith('md5'):
-                        wagon['md5url'] = asset
-                else:
-                    wagon['url'] = asset
-            elif wagon['name'] == CENTOS and 'centos-Core' in asset:
-                if asset.endswith('md5'):
-                        wagon['md5url'] = asset
-                else:
-                    wagon['url'] = asset
+        elif 'centos-altarch' in asset:
```

Comment:
Can't we do something more future-proof here so if we later add more architectures (e.g. manylinux) we won't be tied to fixed indexes in the list?

Reply:
I think not. ATM wagons take the name that we give them in the wagon command. Wagon names are based largely on convention. I can change the name of a wagon just by giving it a different name in a command. So, we can say we are doing "future proofing", but unless we future proof wagon, it is for naught. Also, I highly doubt we are anywhere near manylinux support. Lol.